pandas/io/json/_normalize.py

```diff
@@ -26,12 +27,44 @@ def convert_to_line_delimits(s):
     return convert_json_to_lines(s)
 
 
+def _parse_use_keys(use_keys: Union[str, List, Callable, None]) -> Callable:
+    """
+    Converts different types of use_keys into a callable.
+
+    Parameters
+    ----------
+    use_keys : str, list or callable
+        Returns true or false depending on whether to include or exclude a key.
+
+        .. versionadded:: 1.0.0
+
+    Returns
+    -------
+    callable
+     It Decides on whether to include a key in processing.
+    """
+    if callable(use_keys):
+        return use_keys
+
+    if is_list_like(use_keys):
```

Comment:
these can be if/elseif

Reply:
Not required right? because there is a return at every if.